gradio/queueing.py

```diff
@@ -143,32 +174,45 @@ def _resolve_concurrency_limit(self, default_concurrency_limit):
         else:
             return 1
 
+    def __len__(self):
+        return sum(
```

Comment:
nit: Don't think we need to create a list in memory just to throw it out immediately. A for-loop where we `+=` an variable might be easier to understand.